### DISCUSSION TITLE
[@types/testing-library__jest-dom] Extend the native jest types package

### DISCUSSION THREAD

**Discussion by martin-badin** [DefinitelyTyped/DefinitelyTyped#61312] [Issues with a @types package]
Hi folks. Is it possible to extend the native `@jest/types` package instead of the unofficial `@types/jest` package? Each jest package uses the native types package except this package. I am not sure why it is the reason.

Native package: https://www.npmjs.com/package/@jest/types
Unofficial package: https://www.npmjs.com/package/@types/jest

[a comment by an automated account is not shown]

**Comment by jgoz** [comment]
> Each jest package uses the native types package except this package. I am not sure why it is the reason.

Do you mean "each package that is officially part of Jest" or "each 3rd party package that extends Jest's types"? jest-dom is not an official Jest package, so it would be included in the latter.

> I am not sure why it is the reason.

Jest didn't used to ship its own types, so the only possibility was extending DTS types. It might be possible to use `@jest/types`, but it would likely be a breaking change and might have implications for which versions of Jest are compatible with these types.
